### PR TITLE
HIB-1 Add lambda for populating hyp3 db

### PR DESCRIPTION
**Pull request by Askaholic**
Does not get called from anywhere yet

### CODE REVIEW

File: cloudformation/tropo/templates/hyp3_setup_db.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+from . import utils
+from .hyp3_kms_key import kms_key
+from .hyp3_rds import dbpassword, dbuser, hyp3_db
+
+source_zip = "setup_db.zip"
+
+
+print('adding setup_db lambda')
+
+lambda_name = t.add_parameter(Parameter(
```
Comment (williamh890) [inline cloudformation/tropo/templates/hyp3_setup_db.py:33]: Try and keep parameters cloudformation params to a minimum

File: lambdas/setup_db/src/lambda_function.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+import os
+
+from hyp3_db import Hyp3DB
+from hyp3_db.hyp3_models.base import Base
+from sqlalchemy.sql import text
+
+HOST = os.environ("Hyp3DBHost")
+USER = os.environ("Hyp3DBRootUser")
+PASS = os.environ("Hyp3DbRootPass")
```
Comment (williamh890) [inline lambdas/setup_db/src/lambda_function.py]: Hyp3D**b**RootPass -> Hyp3D**B**RootPass

File: lambdas/setup_db/src/lambda_function.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# setup_db/lambda_function.py
```
Comment (williamh890) [inline lambdas/setup_db/src/lambda_function.py:1]: Most of this should be added to the Hyp3DB class.

**Review by williamh890 — APPROVED** [review]

File: cloudformation/tropo/templates/hyp3_setup_db.py (hunk 1 above)
Comment (Askaholic) [reply cloudformation/tropo/templates/hyp3_setup_db.py:33]: It has a default, so you don't have to specify it

**Pull request merged**